package.json

```diff
@@ -541,7 +541,6 @@
     "post-compile": "npx projen post-compile",
     "post-upgrade": "npx projen post-upgrade",
     "pre-compile": "npx projen pre-compile",
-    "publish:git": "npx projen publish:git",
```

Comment:
🤔

Reply:
Hey, this and the one above are an artifact for the manual release: https://github.com/cdklabs/cdk-cloudformation/pull/361/files#diff-9aedcc318d3b205d9178ea85cafef0e9551122bd5797b08c64b77a4148b98d57R5788-R5809

Reply:
Gotcha!